src/components/carousel/HomepageHeaderCarousel.jsx

```diff
@@ -7,13 +7,17 @@ function HomepageHeaderCarousel({images}) {
     const [autoPlay, setAutoPlay] = useState(true);
     let timeOut = null;
     
-    useEffect(() =>{ // eslint-disable-next-line 
-     timeOut =  autoPlay && setTimeout(() => {
-            slideRight();
-
+    useEffect(
```

Comment:
I think this could cause some issues, just something that Imran mentioned it may be worth checking in with him to get any suggestions

Reply:
yeah will discuss with Imran, not an error at the moment but will look into